operational_analysis/methods/turbine_long_term_gross_energy.py

```diff
@@ -45,12 +45,13 @@ class TurbineLongTermGrossEnergy(object):
     """
 
     @logged_method_call
-    def __init__(self, plant, max_power_filter = 0.85, wind_bin_thresh=2):
+    def __init__(self, plant, max_power_filter = 0.85, wind_bin_thresh=2, rep_thresh=3):
         """
         Initialize turbine long-term gross energy analysis with data and parameters.
 
         Args:
          plant(:obj:`PlantData object`): PlantData object from which TurbineLongTermGrossEnergy should draw data.
+         max_power_filter(float): 
```

Comment:
Add docstring for wind_bin_thresh, rep_thresh, and add description for max_power_filter.